.prettierignore

```diff
@@ -0,0 +1,4 @@
+resources/views/vendor/mail/**/*.blade.php
+resources/views/mail/**/*.blade.php
```

Comment:
Are we doing this preventatively in the event that we create markdown mail?

Reply:
Yeah, exactly, it causes a lot of issues before so just getting out ahead of it just in case it is ever added.

Reply:
This makes sense.